lib/providers/http/services/aens.js

```diff
@@ -125,8 +126,10 @@ class AENS extends HttpService {
    * @param fee
    * @returns {Promise<*>}
    */
-  async update(target, nameHash, {nameTtl = 600000, ttl = 1, fee = 1, privateKey, account}) {
-    if (typeof privateKey !== 'undefined') {
+  async update(target, nameHash, account, options = {}) {
```

Comment:
Might be a seperate later, but we should update the Jdoc when we change the signature.

Reply:
All done